server/channels/app/platform/helper_test.go

```diff
@@ -221,20 +202,33 @@ func setupTestHelper(dbStore store.Store, dbSettings *model.SqlSettings, enterpr
 	}
 
 	err = th.Service.Start(nil)
-	if err != nil {
-		panic(err)
-	}
+	require.NoError(tb, err)
+
+	tb.Cleanup(func() {
+		th.Shutdown(tb)
+	})
 
 	return th
 }
 
-func (th *TestHelper) TearDown() {
-	th.Service.ShutdownMetrics()
-	th.Service.Shutdown()
-	th.Service.ShutdownConfig()
+// Shutdown may be called by tests to manually shut down the [TestHelper].
+// If it's not called manually, it will get called automatically via [testing.TB.Cleanup].
+func (th *TestHelper) Shutdown(tb testing.TB) {
```

Comment:
This is the only place I could find where tests need to shut down the test helper manually. Hence, this helper is around to help with that.